packages/oc-template-handlebars-compiler/lib/resources/strings.js

```diff
@@ -1,10 +0,0 @@
-'use strict';
-
-module.exports = {
-  errors: {
```

Comment:
Let's keep them here for the moment, and perhpaphs have a follow up PR where we move error strings as () => `` in its own package to be shared/reused across templates